applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -345,6 +365,8 @@ static void initialize(le_audio_receive_cb recv_cb)
 	if (!initialized) {
 		receive_cb = recv_cb;
 
+		active_bt_device_name = 0;
```

Comment:
Initialized is never set to false, and active_bt_device_name is a static so it will already be 0.

Reply:
Removed this line

Reply:
So an uninitialized static will always be set to 0?
Is the APP core always clearing statics down to 0 on boot?
This assumption has caused issues with subtle changes in toolkits where the compiler/linker are allowed to only initialise statics if they are explicitly initialised in the code. If they are not then it does nothing and will boot/start-up quicker.
If a static is not initialised with a value it is best not to assume that the memory can be in any state, so I would suggest we keep this.

Reply:
Static variables are always initialized to zero, [C99 Standard (PDF document)](http://www.open-std.org/JTC1/sc22/wg14/www/docs/n1256.pdf):

> If an object that has static storage duration is not initialized explicitly,
then:
— if it has pointer type, it is initialized to a null pointer;
— if it has arithmetic type, it is initialized to (positive or unsigned) zero;
— if it is an aggregate, every member is initialized (recursively) according to these rules;
— if it is a union, the first named member is initialized (recursively) according to these
rules.

This will always be zero when booting